lib/takeaway.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'twilio_sender'
+
+class Takeaway
+
+  attr_reader :menu, :twilio_sender
+  attr_accessor :selection
+
+  def initialize twilio_sender
+    @menu = {
+      pizza: 9.99,
+      chicken: 2.99,
+      kebab: 4.99,
+      beer: 0.99,
+      cider: 0.99,
+      wine: 7.99
+      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.